docs/developer-guide/README.md

```diff
@@ -34,7 +34,7 @@ You're finally ready to start working with rules. You may want to fix an existin
 
 You've developed library-specific rules for ESLint and you want to share it with the community. You can publish an ESLint plugin on npm.
 
-## Section 6: [Working with Parsers](working-with-parsers.md)
+## Section 6: [Working with Parsers](working-with-custom-parsers.md)
```

Comment:
This looks great, but maybe the link text should also say "Working with Custom Parsers"? (I checked the destination page and it does have "Custom" in its title.)

I'm okay with this going in just to fix the broken link, but wanted to leave the suggestion here.